Ensure we link with the just-built libmypaint, not a different version that might already be installed.
https://github.com/mypaint/libmypaint/pull/41
https://github.com/mypaint/mypaint/pull/566
--- brushlib/tests/SConscript.orig	2016-01-15 13:42:55.000000000 -0600
+++ brushlib/tests/SConscript	2016-01-16 13:30:24.000000000 -0600
@@ -19,7 +19,7 @@
     testlib_env.Append(LIBS=['intl'])
 elif sys.platform == "msys" and os.environ.get("MSYSTEM") != "MSYS":
     testlib_env.Append(LIBS=['intl'])
-testlib_env.Append(CPPPATH=['../'], LIBPATH=['../'])
+testlib_env.Prepend(CPPPATH=['../'], LIBPATH=['../'])
 
 if testlib_env['enable_gperftools']:
     parse_pkg_config(testlib_env, "libprofiler")
--- lib/SConscript.orig	2016-01-15 13:42:55.000000000 -0600
+++ lib/SConscript	2016-01-16 13:28:23.000000000 -0600
@@ -21,8 +21,8 @@
 
 # Build against brushlib
 env.Prepend(LIBS=['mypaint-tests', "mypaint"])
-env.Append(LIBPATH="../brushlib")
-env.Append(CPPPATH=['../brushlib', '../brushlib/tests'])
+env.Prepend(LIBPATH="../brushlib")
+env.Prepend(CPPPATH=['../brushlib', '../brushlib/tests'])
 
 
 def parse_pkg_config(env, libname):
